examples/echo_server/include/lwip/lwipopts.h

```diff
@@ -57,19 +109,93 @@
 
 #endif
 
-#define TCP_SND_QUEUELEN 2500
-#define MEMP_NUM_TCP_SEG TCP_SND_QUEUELEN
-#define TCP_SND_BUF (100 * TCP_MSS)
-#define TCP_WND (100 * TCP_MSS)
+/**
+ * TCP Maximum segment size. For the receive side, this MSS is advertised
+ * to the remote side when opening a connection. For the transmit size, this 
+ * MSS sets an upper limit on the MSS advertised by the remote host.
+ */
+#define TCP_MSS 1460
```

Comment:
I thought it seems like a better idea to set this to 1460 since this is actually the highest MSS we can support.